servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -183,13 +183,14 @@ void RenderForwardClustered::RenderBufferDataForwardClustered::clear() {
 	}
 }
 
-void RenderForwardClustered::RenderBufferDataForwardClustered::configure(RID p_color_buffer, RID p_depth_buffer, int p_width, int p_height, RS::ViewportMSAA p_msaa) {
+void RenderForwardClustered::RenderBufferDataForwardClustered::configure(RID p_color_buffer, RID p_depth_buffer, int p_width, int p_height, RS::ViewportMSAA p_msaa, uint32_t p_views) {
```

Comment:
for API consistency I would most likely rename p_views to p_view_count

Reply:
👍

Reply:
Done